pandas/core/series.py

```diff
@@ -2333,16 +2354,25 @@ def combine(self, other, func, fill_value=None):
 
     def combine_first(self, other):
         """
-        Combine Series values, choosing the calling Series's values
-        first. Result index will be the union of the two indexes
+        Combine Series values, choosing the calling Series's values first.
 
         Parameters
         ----------
         other : Series
+            The value(s) to be combined with the `Series`.
 
         Returns
         -------
         combined : Series
```

Comment:
```suggestion
        Series
```

Also add a description of what is being returned.